perf-dashboard/clientlibs-download/python_clientlibs_download.py

```diff
@@ -18,6 +18,8 @@
 
 from google.cloud import bigquery
 
+GCLOUD_PROJECT = 'cloud-python-runtime-qa'
```

Comment:
Now that I'm looking at this, I'd rather use the environment variable or gcloud config info.  As-is, no one outside of Google can run this script because they don't have access to this project (and shouldn't).